src/pages/payment-methods/_mobile-accordian-item.tsx

```diff
@@ -23,8 +23,11 @@ const Notes = styled.div`
     margin: 0 -16px;
     padding: 16px;
 `
-
-const MobileAccordianItem = ({ locale, pd }) => {
+type MobileAccordianItemProps = {
+    locale: object
```

Comment:
The locale property includes the `language` property declared in `ExpandListProps`
It is also not recommended to use the `object` object, because it does not tell us anything else, except that inside we see {}
https://wanago.io/2022/01/24/index-signatures-typescript/